modules/recurlyjs/src/Plugin/Derivative/RecurlyJsLocalTask.php

```diff
@@ -47,7 +58,8 @@ public function __construct(EntityManagerInterface $entity_manager, TranslationI
   public static function create(ContainerInterface $container, $base_plugin_id) {
     return new static(
       $container->get('entity.manager'),
-      $container->get('string_translation')
+      $container->get('string_translation'),
+      $container->get('recurly.config_manager')
```

Comment:
I wish you could add commas after the last argument so they were consistent with multiline arrays.